fmriprep/workflows/bold/base.py

```diff
@@ -826,7 +826,7 @@ def init_func_preproc_wf(bold_file):
     # Fill-in datasinks of reportlets seen so far
     for node in workflow.list_node_names():
         if node.split('.')[-1].startswith('ds_report'):
-            workflow.get_node(node).inputs.base_directory = reportlets_dir
+            workflow.get_node(node).inputs.base_directory = output_dir
```

Comment:
This will also put HTML snippets in `figures/`, right?

Reply:
Correct

Reply:
Do we want that?

Reply:
Well, what's the harm?

On the pro side - when I wrote the sMRIPrep fast-track I found an issue in generating the T1w summary (because the actual inputs were not available anymore)(*). So I had to set up an alternative that says: "the T1w was reused from a previous run".

Now, we could remove that alternative, as you indeed have the HTML excerpt to integrate in the report.

(*) Actually, this stands for any of the reportlets - there's a fair amount of logic in smriprep to regenerate all the reportlets if you are on the fast track.

Reply:
Okay.